Main/ClientApp/src/Components/Drawer/DrawerItemNested.tsx

```diff
@@ -31,21 +37,27 @@ export default function DrawerItemNested(props: DrawerItemNestedProps) {
   };
   const onMenuClose = () => setAnchorEl(null);
   return (
-    <ListItem button className={classes.nested}>
-      {props.icon != null ? (
-        <ListItemIcon children={props.icon} />
-      ) : (
-        <ListItemIcon children={<Clear style={{ visibility: "hidden" }} />} />
-      )}
-      <ListItemText primary={props.title} />
-      {/*
+    <>
+      <ListItem button className={classes.nested}>
+        <Link to={props.link} className={classes.link}>
```

Comment:
Link here completely destroys list item view; icon & text are stacked on top of each other instead of being horizontally spaced; fix needed

Reply:
Not completely, it is a small issue to be fixed with layout change. It is better to have this small UI bug than having console errors thrown all the time